plainly-aescripts/src/project.ts

```diff
@@ -9,9 +9,13 @@ function setProjectData(id: string, revisionCount: number) {
     ExternalObject.AdobeXMPScript = new ExternalObject('lib:AdobeXMPScript');
   }
 
-  const mdata = new XMPMeta!(project.xmpPacket); //get the project's XMPmetadata
+  if (!XMPMeta) {
+    return 'Error: Project metadata functionality is unavailable';
```

Comment:
Any reason why we add this "Error: " to the message?

I think we should `throw new Error(message)`. 

What functionality won't work if `XMPMeta` is not available? I think we should state that, example:

`throw new Error("Unable to link the project due to missing metadata")`